src/object-field/object-field.component.html

```diff
@@ -21,7 +21,7 @@
   <!-- ADD SUB FIELD FROM SCHEMA DROPDOWN -->
   <tr>
     <td class="button-holder">
-      <add-field-dropdown [fields]="keys" [pathString]="pathString" (onFieldAdd)="onFieldAdd($event)" [schema]="schema.properties">+</add-field-dropdown>
```

Comment:
I also unified the `@Input() schema`, `add-field-dropdown` was taking `schema.properties` as input and that might be a bit confusing later, it's better to be able to assume that when you see `schema`, it's JSONSchema 😄